src/components/Layout.tsx

```diff
@@ -0,0 +1,50 @@
+import { theme } from "@/theme";
+import { MdOutlineDarkMode, MdOutlineLightMode } from "react-icons/md";
+import { ColorMode, Flex, Icon, ThemeProvider, ToggleButton, ToggleButtonGroup, View } from "@aws-amplify/ui-react";
+import React from "react";
+
+function AppHeader({ children }: React.PropsWithChildren) {
+  return (
+    <Flex as="header" direction="row" justifyContent="space-between" alignItems="center" padding="1rem" boxShadow="small" position="sticky" top="0" left="0" width="100%" backgroundColor="background.primary">
+      {children}
+    </Flex>
+  );
+}
+
+
+export default function Layout({ children }: React.PropsWithChildren) {
+  const [colorMode, setColorMode] = React.useState<ColorMode>("light");
+  return (
+    <ThemeProvider theme={theme} colorMode={colorMode}>
```

Comment:
Just need the ThemeProvider at the top of the render tree. If you want to use the theme somewhere you can just use `useTheme()`

Reply:
Good to know!